src/klassy.js

```diff
@@ -0,0 +1,12 @@
+import bem from 'react-bem-helper';
+
+export default function klassy (packageJSON, override) {
```

Comment:
Classnames are useless here yet. There's a `react-bem-helper` which does basically the same.